### PR TITLE
User Profile Card Component

### PR DESCRIPTION
**Pull request by LHBO19**
Issue #28 

Created a User Profile Card that has the user contact info and a placeholder button. Added the component to Profile View.

### CODE REVIEW

**Comment by LHBO19** [comment]
[User profile card component](https://trello.com/c/sUzRhr7y/32-user-profile-card-component)

File: src/Components/UserProfileCard.js [hunk 1]
```diff
@@ -1,25 +1,40 @@
 import React from 'react';
-import { Button, Card, Icon, Image } from 'semantic-ui-react';
+import { Grid, Button, Card, Icon, Image } from 'semantic-ui-react';
 
 const UserProfileCard = () => {
   return (
     <Card centered>
```
Comment (shibeknight) [inline src/Components/UserProfileCard.js]: The card here needs to have the fluid prop so it takes the full space of the container.

File: src/Components/UserProfileCard.js [hunk 2]
```diff
@@ -1,25 +1,40 @@
...
-        </Card.Meta>
-        <Card.Description>Arturo is a musician living in CDMX.</Card.Description>
-
-        <Button centered color="blue" type="submit">
-          Message
-        </Button>
+        <Grid>
+          <Grid.Column mobile={16} tablet={6} computer={8}>
+            <Grid.Row>
+              <Image
```
Comment (shibeknight) [inline src/Components/UserProfileCard.js:11]: Try to give the image a circular shape by adding radius or looking into semantic's docs to see if the Image component has a prop for that.

File: src/Components/UserProfileCard.js [hunk 3]
```diff
@@ -1,25 +1,40 @@
...
+            </Grid.Row>
+            <Grid.Row centered>
+              <Button centered color="blue" type="submit">
+                Message
+              </Button>
+            </Grid.Row>
+          </Grid.Column>
+        </Grid>
       </Card.Content>
       <Card.Content extra>
```
Comment (shibeknight) [inline src/Components/UserProfileCard.js:39]: After you resize the card by making it fluid try to add one or two items to the card.content extra and see how it looks, just to make it look more similar to the reference layout.

**Review by shibeknight — DISMISSED** [review]

**Review by shibeknight — DISMISSED** [review]

**Review by shibeknight — APPROVED** [review]

File: src/Components/UserProfileCard.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            </Grid.Row>
+          </Grid.Column>
+          <Grid.Column mobile={16} tablet={6} computer={6}>
+            <Grid.Row centered>
+              <Card.Header>Arturo</Card.Header>
+              <Card.Meta>
+                <span className="position">CEO</span>
+              </Card.Meta>
+              <Card.Description>
+                <span className="email">Avelo@gmail.com</span>
```
Comment (avelinoschz) [inline src/Components/UserProfileCard.js:25]: Great placeholder :+1:

**Review by avelinoschz — APPROVED** [review]

**Pull request merged**